service/routes.py

```diff
@@ -160,3 +160,9 @@ def abort_with_error(error_code, error_msg):
     """
     app.logger.error(error_msg)
     abort(error_code, error_msg)
+    
```

Comment:
@mingyi850 has different implementations for handling the internal server error in this PR(https://github.com/CSCI-GA-2820-SU24-001/promotions/pull/19/files), I think we need to agree on one consistent approach for our error handling. Could you two align on this?